rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -36,7 +40,12 @@
 
 
 class SQLStorageManager(object):
+    @traces()
```

Comment:
Why do u need to use 'traces' when u are init-ing it?

Reply:
It is a function that is being called, so thought it's something useful. Should I just remove it then?

Reply:
I'll rename the enable_tracing function, that enables tracing of the SQL not Python execution tracing.

Reply:
Is that OK?